tests/qos/test_buffer_traditional.py

```diff
@@ -0,0 +1,222 @@
+import logging
+
+import pytest
+
+from tests.common.utilities import wait_until
+from tests.common.helpers.assertions import pytest_assert
+
+def load_lossless_info_from_pg_profile_lookup(duthost):
+    """Load pg_profile_lookup.ini to a dictionary. Called only once when the module is initialized
+
+    Args:
+        duthost: the DUT host object
+
+    Return:
+        The dictionary containing the information in pg_profile_lookup.ini
+    """
+    # Check the threshold mode
+    threshold_mode = duthost.shell('redis-cli -n 4 hget "BUFFER_POOL|ingress_lossless_pool" mode')['stdout']
+    threshold_field_name = 'dynamic_th' if threshold_mode == 'dynamic' else 'static_th'
+    dut_hwsku = duthost.facts["hwsku"]
+    dut_platform = duthost.facts["platform"]
+    skudir = "/usr/share/sonic/device/{}/{}/".format(dut_platform, dut_hwsku)
+    lines = duthost.shell('cat {}/pg_profile_lookup.ini'.format(skudir))["stdout"]
```

Comment:
please test for existence of this file before doing the cat

Reply:
Fixed